src/provider/rpc.ts

```diff
@@ -648,7 +640,7 @@ export class RpcProvider implements ProviderInterface {
 
   /**
    * Returns all events matching the given filter
-   * @returns RPC.Events - events and the pagination of the events
```

Comment:
Here i left type because for some reason response type shown is Promise<RPC.SPEC.EVENTS_CHUNK> instead of Promise<RPC.Events>

Reply:
The notation for the return type should use braces e.g. `@return {RPC.Events}`, however it would also be ignored. Not sure if there is a way to force TS to use the `RPC.Events` alias and not the base type it unspools to.

Reply:
Yea not sure, we can go back to this question in calmer times.